scripts/db.univar/db.univar.py

```diff
@@ -313,23 +313,23 @@ def main():
         pval[i] = 0
 
     inf = open(tmp + ".sort")
-    l = 1
+    ln = 1
```

Comment:
Is there a better name for this one? What was the reasoning for it (does it come from a formula or something)?

I'm reading it like the natural logarithm (ln), like in maths, but doesn't make sense.

Reply:
The "line_number" is what I meant.  The logarithm didn't occur to me, but you're right. Would "line_number" or "counter" make sense?

Reply:
line_number sounds better than counter for me.

Reply:
Also, a note for other readers: I don't know if this part of code is critical or not, but that loop and double loop is probably quite inefficient. Probably doing a "in" check (with a pre-created local tuple) once per loop, and in the 4+pops times it matches, then find the correct assignment.

It's not something to do with this PR.